act/routing.md

```diff
@@ -0,0 +1,60 @@
+# Routing
+
+## Rule Identifier
+
+gel-components-routing
+
+## Rule Description
+
+Routing is the behaviour of single page applications to emulate the coventions of page loads.
+
+## Rule Type
+
+atomic
+
+## Accessibility Requirements Mapping
+
+- [BBC GEL Routing](https://bbc.github.io/gel/foundations/routing/)
+  - Required for conformance to BBC GEL
+  - Outcome mapping:
+    - Any failed outcomes: success criterion is not satisfied
+    - All passed outcomes: success criterion needs further testing
+    - An inapplicable outcome: success criterion needs further testing
+
+## Input Aspects
+
+* DOM Tree
+
+## Applicability
+
+This rule applies to any HTML page that includes a BBC GEL x component.
+
+## Expectations
+
+### Expectation 1
+
+The document `<title>` element content must reflect the current screen at all times. It must described the purpose of the screen and be unique within the screens of the website or appliction.
```

Comment:
We had this debate in MAG recently, regarding that word "unique". For a large website like the BBC, that seems like a tall order. I think we will end up saying that the title must be descriptive and distinctive rather than "unique".